vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopVerifier.kt

```diff
@@ -296,7 +336,7 @@ class OidcSiopVerifier private constructor(
     ): KmmResult<JwsSigned> = catching {
         val requestObject = createAuthnRequest(requestOptions)
         val requestObjectSerialized = jsonSerializer.encodeToString(
-            requestObject.copy(audience = relyingPartyUrl, issuer = relyingPartyUrl)
+            requestObject.copy(audience = "https://self-issued.me/v2", issuer = "https://self-issued.me/v2")
```

Comment:
Why use those strings as default values?

Reply:
https://openid.net/specs/openid-connect-self-issued-v2-1_0.html#name-aud-of-a-request-object and https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-aud-of-a-request-object